src/forges/gitea.ts

```diff
@@ -37,32 +46,52 @@ export class GiteaForge extends Forge {
     sourceBranch: string;
     targetBranch: string;
   }): Promise<{ pullRequestLink: string }> {
-    const pullRequest = await this.handleApiErrors(
-      this.api.repos.repoGetPullRequestByBaseHead(
-        options.owner,
-        options.repo,
-        options.targetBranch,
-        options.sourceBranch,
-      ),
+    const pullRequestAll = await this.handleApiErrors(
+      this.api.repos.repoListPullRequests(options.owner, options.repo, { state: 'all' }),
     );
 
-    if (pullRequest?.data && pullRequest.data.state === 'open') {
-      const pr = await this.api.repos.repoEditPullRequest(options.owner, options.repo, pullRequest.data.number!, {
-        title: options.title,
-        body: options.description,
-      });
+    const filteredPullRequests = pullRequestAll?.data?.filter(
+      (prList) =>
+        prList.base?.ref === options.targetBranch &&
+        prList.head?.ref === options.sourceBranch &&
+        prList.state === 'open' &&
+        prList.title === options.title,
+    );
 
-      return { pullRequestLink: pr.data.html_url! };
+    let pullRequest = filteredPullRequests?.[0];
```

Comment:
Might be helpful to warn the user in case we get more than one PR back

Reply:
As this happens after the successful merge of the first release, I am not sure this should be a persistent warning?

Reply:
Ah, also this can't ever be > 1 as there can only ever be one open PR with this specific base and head?